projects/components/src/utils/common-util.spec.ts

```diff
@@ -15,4 +16,29 @@ describe('CommonUtil', () => {
     it('can round to 0 when no number is given', () => {
         expect(CommonUtil.roundTo(null)).toEqual(0);
     });
+
+    describe('createBufferedPromise', () => {
+        it(
```

Comment:
Prettier should allow you to have longer lines in the description of `it` the broken lines make it harder to read.

Here's a suggestion

`it returns a function that does not call the passed in function until x ms have elapsed since the returned function was called`